netbench/cdk/src/main/java/com/aws/EcsStack.java

```diff
@@ -153,6 +162,7 @@ public EcsStack(final Construct parent, final String id, final EcsStackProps pro
                 .logging(LogDriver.awsLogs(AwsLogDriverProps.builder().logGroup(serviceLogGroup).streamPrefix(stackType + "-ecs-task").build()))
                 .portMappings(List.of(PortMapping.builder().containerPort(3000).hostPort(3000)
                     .protocol(software.amazon.awscdk.services.ecs.Protocol.UDP).build()))
+                .privileged(true)
```

Comment:
why is this needed?

Reply:
Oh this was for testing bpftrace in my Docker containers, I'll remove it since I don't need it anymore.